### DISCUSSION TITLE
add persons to an order and print the new ticket

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3209] [Q&A] [answered]
Hello Pretix Team!

I have a case where for an event created, that someone can buy online let's say 1 ticket for 10 persons and when he/she comes to the Box office needs to add more persons in the same order.
Can the pretixPOS add more persons to the same order ? Or this is something that can be done from the web interface only ?
If this is doable only from the web interface, how can the new ticket be printed ? From the web interface or the pretixPOS can see the new ticket and print it ?

Thank you in advance

**Comment by raphaelm** [comment, marked as answer]
>  Can the pretixPOS add more persons to the same order ? Or this is something that can be done from the web interface only ?

Web interface only

> If this is doable only from the web interface, how can the new ticket be printed ? From the web interface or the pretixPOS can see the new ticket and print it ?

You can download the PDF from the webinterface, or go back to pretixPOS, as you prefer
